doc/Configurations.md

```diff
@@ -156,6 +156,8 @@ You can see the hash by typing `IRB.conf`. Below are the primary entries:
 - `:USE_TRACER`: Whether to use the
   {IRB tracer}[rdoc-ref:IRB@Tracer];
   initial value: `false`.
+- `:USE_PAGER`: Controls whether "pagination" is enabled.
```

Comment:
How about using the word `pager` instead of quoted `"pagination"`?
This document file already use the word `pager`
```md
- `RI_PAGER` / `PAGER`: Specifies the pager for documentation.
```

I think the word pagination is mostly used for dividing content into multiple numbered pages (https://en.wikipedia.org/wiki/Pagination)
https://en.wikipedia.org/wiki/Terminal_pager does not use the word pagination.

Reply:
That's a great idea! I didn't know the terminal pager is a thing. Thank you for enlightening!